driver_nuopc/ice_import_export.F90

```diff
@@ -1790,10 +1790,10 @@ subroutine ice_prescribed_init(clock, mesh, my_task, master_task, nu_diag, domai
     integer, pointer :: model_year_align    ! align stream_year_first
                                    ! with this model year
 !DD    character(len=strKIND), pointer :: stream_fldVarName
```

Comment:
Just wanted to check before I approve, is there a reason why `512` is being hard-coded instead of using `strKIND`? My understanding is `strKIND` should be the preferred and more portable way of doing things in MPAS codes.